### DISCUSSION TITLE
purchased crated supplies disappear after round restart

### DISCUSSION THREAD

**Discussion by doganesian** [FakeFishGames/Barotrauma#13836] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If you buy supplies to be delivered to your submarine, and you return to server lobby in multiplayer. Restarting from the save voids the crates.

In the past, doing this kept the crates.

### Reproduction steps

1. Start a multiplayer campaign
2. Purchase supplies and choose deliver to submarine
3. Start a mission, (the supplies are correctly delivered)
4. Return to server lobby
5. Load campaign from save (all missions and everything else is saved properly)
6. You will not have the purchased crate

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by AzureNocte** [comment]
I think it is addressed in [#13820](https://github.com/FakeFishGames/Barotrauma/issues/13820)

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13819
